library/Reporting/Reportlet.php

```diff
@@ -6,33 +6,38 @@
 
 class Reportlet
 {
-    /** @var int */
-    protected $id;
-
     /** @var string */
     protected $class;
 
     /** @var array */
     protected $config;
 
     /**
-     * @return  int
-     */
-    public function getId()
-    {
-        return $this->id;
-    }
-
-    /**
-     * @param int $id
+     * Transform the reportlet model to this class
```

Comment:
Since this is a factory function I would write something like `Create reportlet from the given model`.